src/rust/ide/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -429,6 +429,15 @@ impl ContainerModel {
         // self.view.background.shape.roundness.set(value);
         // self.fullscreen_view.background.shape.roundness.set(value);
     }
+
+    /// Check if given mouse-event-target means this visualization.
+    fn is_this_target(&self, target:scene::Target) -> bool {
+        use ensogl::display::shape::primitive::system::Shape;
+        self.view.overlay.shape.sprites().iter().map(|s| scene::Target::Symbol {
+            symbol_id   : s.symbol.id as u32,
+            instance_id : *s.instance_id as u32,
+        }).any(|t| t == target)
+    }
```

Comment:
this could be refactored to soemthing more generic for any sprite system. And I believe it should, its very very useful utility.